packages/launchpad/src/types.ts

```diff
@@ -54,12 +54,14 @@ export interface LaunchpadResponse {
 	checklist_statuses: LaunchpadStatuses[];
 }
 
+export interface PermittedActions {
```

Comment:
Nit: Might it be clearer to specify that this is launchpad-specific?
```suggestion
export interface PermittedLaunchpadActions {
```